server/src/main/java/ru/hse/server/utils/ValidationStep.java

```diff
@@ -0,0 +1,26 @@
+package ru.hse.server.utils;
+
+public abstract class ValidationStep<T> {
```

Comment:
can make this class internal/protected

Reply:
Don't understand what is internal class, is it package private, if so i want to reuse this class so i think better to leave as it
Also my class can't be protected since it is abstract